redaxo/src/addons/structure/package.yml

```
@@ -20,6 +20,8 @@ pages:
         hidden: true
         perm: structure/hasStructurePerm
         pjax: true
+        subpages:
+            - linkmap: { title: translate:linkmap }
```

```suggestion
            linkmap: { title: translate:linkmap }
```

Das war wahrscheinlich die Ursache für das Problem.